nltk/data.py

```diff
@@ -72,7 +72,8 @@
    (e.g., in their home directory under ~/nltk_data)."""
 
 # User-specified locations:
-path += [d for d in os.environ.get('NLTK_DATA', str('')).split(os.pathsep) if d]
+path += [d for d in os.environ.get('NLTK_DATA',
+                                   str('')).split(os.pathsep) if d]
```

Comment:
I think it is not good to split the line at this point - it is not clear which expression is an argument of which function. Add a local variable, e.g like this:

```
_paths_from_env = os.environ.get('NLTK_DATA', str('')).split(os.pathsep)
path += [d for d in _paths_from_env if d]
```